tests/sentry/tasks/test_derive_code_mappings.py

```diff
@@ -269,11 +248,10 @@ def test_derive_code_mappings_single_project(
         assert code_mapping.exists()
         assert code_mapping.first().automatically_generated is True
 
-    def test_skips_supported_projects(self):
-        new_data = deepcopy(self.test_data)
-        new_data["platform"] = "elixir"
-        event = self.store_event(data=new_data, project_id=self.project.id)
-        assert derive_code_mappings(self.project.id, event.data) is None
+    def test_skips_not_supported_platforms(self):
+        data = self.generate_data([{}], platform="elixir")
```

Comment:
The base class has this function which simplifies saving an event for a test.

Reply:
TIL!

Reply:
> TIL!

FYI This is not part of `TestCase` but the class I created in this module.